__tests__/src/components/TicketsTable.test.tsx

```diff
@@ -97,4 +97,92 @@ describe('TicketsTable', () => {
     render(<TicketsTable items={[]} />);
     expect(screen.getByText('Urgent')).toBeInTheDocument();
   });
+
+  it('renders Urgent on table if a case is Urgent', () => {
+    render(<TicketsTable items={[]} />);
+    expect(screen.getByText('Urgent')).toBeInTheDocument();
+  });
+
+  it('should sort the table when column headers are clicked', () => {
+    const mockItemsSort: ServiceRequestSummary[] = [
```

Comment:
Can you get these mocks from the TestData.ts file?

Reply:
I spent hours trying to test it with testdata the value changes. When Isolating a column and pressing the sorting sometimes is Alison, next set it Brandon, then Charlie as being the first element after the header that it expects.  It's a lot easier to test this way. I need some explanation or at least see the values. I have tried console logging it all I get are big objects with no distinctive values.

Reply:
Done